demos/apps/bookinfo/README.md

```diff
@@ -136,7 +136,7 @@ route requests to all available versions of a service in a random fashion.
    You can display the routes that are defined with the following command:
 
    ```bash
-   $ istioctl list route-rule
+   $ istioctl get route-rule
```

Comment:
We can't merge this PR without also updating the istioctl binaries in istio/demos, since this will break the demo.